po/perform_translations/README.md

```diff
@@ -0,0 +1,14 @@
+This python script helps generate a translated template to work on to speed up the translation workflow. It generates a file $language.po.new that you can fix up and then rename to $language.po.  In order to use this translator template generator you will want to run this to install the "polib" and "dl-translate" modules. Make sure GNU gettext utilities are installed!
+
+
+```
+pip3 install polib dl-translate
+
+```
+
+
```

Comment:
You might want to give an example of how to actually call the script? Ie, what commands you need to run before the script, from what directory you run it, with what options, etc.